Migration.md

```diff
@@ -1,4 +1,4 @@
-## Ignition Fuel Tools 5.X to 6.X
+## Ignition Fuel Tools 7.X to 8.X
```

Comment:
We should add a new section, I think the previous content actually belonged to 5->6